### ISSUE TITLE
为同一个设备的不同通道设置波特率

### ISSUE THREAD

**Issue by Memo654** [hardbyte/python-can#1548]
### Is your feature request related to a problem? Please describe.

目前，Bus类的接口中channel支持list/tuple类型，但bitrate类型为int， 这样就会造成打开多个通道时，所有通道的波特率相同。
我想对同一个Bus的不同的通道配置不同的波特率
![image](https://user-images.githubusercontent.com/59411181/228331191-c0c6ddc8-776a-4ff3-8557-03d079c0b40f.png)
### Describe the solution you'd like

我认为bitrate接口的数据类型可以扩展支持list/tuple类型，并于channel对齐
### Describe alternatives you've considered

无
### Additional context

无

### Note
我没有检索到有类似的问题，如果有重复，请告诉我，我会关闭这个问题。

**Comment by Memo654** [comment]
@zariiii9003 Ok, I update the comment.

**Comment by zariiii9003** [comment]
You could create multiple `CanalystIIBus` instances with different bitrates. While i see, that it could be useful to accept Sequences for bitrate, that would be quite a big change.

**Comment by Memo654** [comment]
Yeah, It looks like a big chanage. I will change some code when I go off work. Hope it will be helpful in the future.
